package.json

```diff
@@ -56,7 +56,10 @@
     "test:a11y": "echo test:a11y",
     "test:smoke": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run --steps -c test/functional/codecept.conf.js --grep @smoke --reporter mochawesome",
     "test:functional": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run --steps -c test/functional/codecept.conf.js --grep @preview --reporter mochawesome",
-    "test:fullfunctional": "NODE_TLS_REJECT_UNAUTHORIZED=0 TESTS_FOR_ACCESSIBILITY='false' codeceptjs run --steps -c test/functional/codecept.conf.js --grep @nightly --reporter mochawesome",
+    "test:fullfunctional": "yarn playwright install && NODE_TLS_REJECT_UNAUTHORIZED=0 TESTS_FOR_ACCESSIBILITY='false' codeceptjs run --steps -c test/functional/codecept.conf.js --grep @nightly --reporter mochawesome",
+    "test:functional-chromium": "yarn playwright install chromium && yarn playwright install firefox && yarn playwright install webkit && yarn playwright test --project=chromium",
```

Comment:
Do you need to install chromium and firefox and webkit when running just for one of them?